R/jhu_data.R

```diff
@@ -28,37 +34,58 @@
 load_jhu_data <- function(
     issue_date = NULL,
     as_of = NULL,
+    location_code = NULL,
     spatial_resolution = "state",
     temporal_resolution = "weekly",
     measure = "deaths",
+    hub = c("US", "ECDC"),
     replace_negatives = FALSE,
     adjustment_cases = "none",
     adjustment_method = "none") {
   # validate measure and pull in correct data set
   measure <- match.arg(measure, choices = c("cases", "deaths"))
   
   # validate issue_date and as_of and load preprocessed data
-  jhu_data <- preprocess_jhu_data(issue_date, as_of, measure)
-
-  # validate spatial_resolution
-  spatial_resolution <- match.arg(
-    spatial_resolution,
-    choices = c("county", "state", "national"),
-    several.ok = TRUE
-  )
+  jhu_data <- preprocess_jhu_data(issue_date, as_of, measure, hub)
+  
+  if (hub[1] == "US"){
+    valid_locations <- covidData::fips_codes
+  } else if (hub[1] == "ECDC"){
+    valid_locations <- covidData::ecdc_locations
```

Comment:
Instead of ecdc_locations, let's use some other list that includes all valid countries (not just Europe)